app/views/admin/settings/index.html.slim

```diff
@@ -46,25 +50,26 @@ section.block
           / this will display a button that the user clicks to start the flow
           ipp:connectToIntuit
 
-- if @accounts && !@accounts.empty?
-  section.block
-    h2 = t("quickbooks.accounts.accounts")
-    p.help-block.note = t("quickbooks.accounts.note")
+- if @division.quickbooks_connected? && @fetch_task && @fetch_task.succeeded?
```

Comment:
Only display accounts form once full fetch has succeeded.

Reply:
This change is in latest commit; easy to take out if we should leave as it was for now (accounts form visible when there's the division has a  connection saved in db)